### PR TITLE
Compatibility with Rails 5.0

### PR DESCRIPTION
**Pull request by pschambacher**
/cc @zendesk/zendesk-rails-upgraders 

mind blown

 * Removing alias_method_chain in favour of prepend
 * Removing attr_accessible (protected_attributes won't be compatible with Rails 5.0)
 * Fixing some ActiveRecord version checks

### CODE REVIEW

File: gemfiles/rails5.0.gemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+source "https://rubygems.org"
+
+gemspec :path=>"../"
```
Comment (grosser) [inline gemfiles/rails5.0.gemfile]: 1.9 syntax ?

File: gemfiles/rails5.0.gemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+gem "activerecord", git: 'git@github.com:rails/rails.git', ref: 'master'
```
Comment (grosser) [inline gemfiles/rails5.0.gemfile]: ref master can be left out ?

File: lib/property_sets/active_record_extension.rb [hunk 3]
```diff
@@ -173,14 +173,14 @@ def self.included(base)
         base.alias_method_chain :update_attributes!, :property_sets
```
Comment (grosser) [inline lib/property_sets/active_record_extension.rb]: remove this ?

**Comment by grosser** [comment]
:+1:

File: gemfiles/rails5.0.gemfile (hunk 2 above)
Comment (pschambacher) [inline gemfiles/rails5.0.gemfile]: Maybe

File: gemfiles/rails5.0.gemfile (hunk 1 above)
Comment (pschambacher) [inline gemfiles/rails5.0.gemfile]: Yeah let's get wild

File: lib/property_sets/active_record_extension.rb (hunk 3 above)
Comment (pschambacher) [inline lib/property_sets/active_record_extension.rb]: Oh yeah good point, I missed this line

**Comment by pschambacher** [comment]
@grosser JRuby doesn't seem to be happy with the prepend...
Also can't git clone on Travis.. I think it's missing an SSH key

**Comment by grosser** [comment]
it's a public gem oO

**Comment by pschambacher** [comment]
I know... but the clone is failing. Haven't had time to dig dipper yet

**Comment by pschambacher** [comment]
@grosser I was getting a lot of errors with JRuby (change from JRuby 1.7 to JRuby 9000 and all that stuff). Since we're not using it I thought about dropping support and then thought it might be time to get a new major version and drop Rails 3.2?
If we really need something we can still branch from the tag and release a patch

**Comment by pschambacher** [comment]
that said the git clone of active record and arel still fails...

**Comment by grosser** [comment]
I'd prefer to keep 3.2 unless we can clean out a lot of crappy code when we
drop it ...

On Thu, Jun 4, 2015 at 7:57 AM, Pierre Schambacher <notifications@github.com

**Comment by bquorning** [comment]
Upgraded to Rails 5.0.0.beta1, rebased on master and force-pushed. Sorry for intruding :-)

File: lib/property_sets/active_record_extension.rb (hunk 3 above)
Comment (grosser) [inline lib/property_sets/active_record_extension.rb]: remove this ?

**Pull request closed without merging**